modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -62,6 +62,9 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 //         health_timeout <duration>
 //         health_status <status>
 //         health_body <regexp>
+//         health_headers {
+//             <field> <value>
```

Comment:
```suggestion
//             <field> [<values...>]
```

Since the value is technically optional and there may also be more than one.

Reply:
ok